events/tests/test_api.py

```diff
@@ -2670,10 +2674,12 @@ def test_import_ticket_system_passwords_errors_with_integrity_errors(
     )
     # Passwords should contain only the added passwords
     assert executed["data"]["importTicketSystemPasswords"]["passwords"] == new_passwords
-    errors_as_text = str(
-        executed["data"]["importTicketSystemPasswords"]["errors"]["passwords"]
+    assert all(
+        [
+            p in str(executed["data"]["importTicketSystemPasswords"]["errors"])
+            for p in existing_passwords
+        ]
```

Comment:
I'd suggest not using `p in str(...)` here as there are more type safe and correct ways, e.g. something like (Didn't actually try if this works):
```python
error_values = [error["value"] for error in executed["data"]["importTicketSystemPasswords"]["errors"]]
assert all(p in error_values for p in existing_passwords)
assert all(p not in error_values for p in new_passwords)
```

Reply:
it will have a different format when I get the damn CI to handle the snapshots the same way as my local env does.